docs/services/minecraft.md

```diff
@@ -0,0 +1,90 @@
+<!--
+SPDX-FileCopyrightText: 2025 XHawk87
+
+SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+# Minecraft
+
+[Minecraft](https://docker-minecraft-server.readthedocs.io) is a first-person open-world procedurally-generated voxel-based sandbox game with RPG elements. This role allows you to self-host your own systemd-managed server in a docker container.
+
+Note: itzg docker-minecraft server is published under the Apache-2.0 license, however Minecraft itself is proprietary software, and by using this role you are agreeing to the [EULA](https://www.minecraft.net/en-us/eula). Know your rights!
+
+
+## Dependencies
+
+This service currently requires no other services.
+
+
+## Configuration
+
+To enable this service, add the following configuration to your `vars.yml` file and re-run the [installation](../installing.md) process:
+
+```yaml
+########################################################################
+#                                                                      #
+# minecraft                                                            #
+#                                                                      #
+########################################################################
+
+minecraft_enabled: true
+minecraft_port: 25565
+minecraft_docker_version: latest
+minecraft_container_volumes_additional:
+  - /run/screen/S-ubuntu:/run/screen/S-{{ minecraft_identifier }}
+minecraft_environment_variables_additional: |
+  MOTD=[Your Server Name Here]
+  TYPE=PAPER
+  INIT_MEMORY=500M
+  MAX_MEMORY=2G
+  USE_AIKAR_FLAGS=true
+  STOP_SERVER_ANNOUNCE_DELAY=10
+  TZ=Europe/London
+  LOG_TIMESTAMP=true
+  DIFFICULTY=hard
+  ENABLE_WHITELIST=true
+  ENFORCE_WHITELIST=true
+  WHITELIST=[Your Minecraft IGN Here]
+  ENABLE_QUERY=false
+  MAX_PLAYERS=20
+  ENABLE_COMMAND_BLOCK=true
+  SNOOPER_ENABLED=false
+  SPAWN_PROTECTION=0
+  VIEW_DISTANCE=16
+  PVP=true
+  ALLOW_FLIGHT=TRUE
+  SIMULATION_DISTANCE=16
+  PLAYER_IDLE_TIMEOUT=0
+  ENABLE_AUTOPAUSE=true
+  MAX_TICK_TIME=-1
+  JVM_DD_OPTS=disable.watchdog:true
+  AUTOPAUSE_TIMEOUT_EST=30
+  AUTOPAUSE_TIMEOUT_INIT=30
+  AUTOPAUSE_TIMEOUT_KN=30
+  INITIAL_ENABLED_PACKS=vanilla,bundle
+
+########################################################################
+#                                                                      #
+# /minecraft                                                           #
+#                                                                      #
+########################################################################
+```
+
+In the example configuration above, we configure the server to be hosted at `mash.example.com:25565`. It will be the latest Paper minecraft server with 2GB of RAM, hard difficulty, only you on the whitelist, rather generous render distance, PvP, bundles, and an automatic server pause when the last player leaves.
```

Comment:
```suggestion
In the example configuration above, we configure the server to be hosted at `mash.example.com:25565`. It will be the latest Paper Minecraft server with 2GB of RAM, hard difficulty, only you on the whitelist, rather generous render distance, PvP, bundles, and an automatic server pause when the last player leaves.
```

An anchor link for "Paper Minecraft server" would be appreciated :slightly_smiling_face: